packages/cli/src/lib/snapshot/reportViewer.ts

```diff
@@ -27,6 +27,7 @@ export class ReportViewer {
     green: (txt: string) => green(txt),
     yellow: (txt: string) => yellow(txt),
     red: (txt: string) => red(txt),
+    gray: (txt: string) => gray(txt),
```

Comment:
I think gray is good 🤔 

I've asked people in another channel and that was their first instinct. Maybe just pick complimentary color for all of them ? It could be easier on the eye.

example: https://getbootstrap.com/docs/5.0/components/alerts/

Suggestion: Instead of pure `green/yellow/red` use those hex.

Reply:
I tried Bootstrap hex colors but it doesn't look too good in a dark terminal.
It's either too bright or too dark. 

Chalk default colors (green, yellow, red) seem to be widely used across other packages.